[RHACS] ROX-25421: Document troubleshooting for secured cluster upgrader (#80296)
openshift/openshift-docs: upgrading/upgrade-roxctl.adoc

====
* If you are using automatic upgrades:
** Update all your secured clusters by using automatic upgrades.

> ** For troubleshooting the problems with the automatic cluster upgrader refer to the xref:../upgrading/upgrade-roxctl.adoc#troubleshooting_upgrader_{context}[Troubleshooting guide]

Reviewer: ```suggestion
** For information about troubleshooting problems with the automatic cluster upgrader, see xref:../upgrading/upgrade-roxctl.adoc#troubleshooting_upgrader_{context}[Troubleshooting the cluster upgrader].
```
